scripts/build-package.sh

```diff
@@ -23,6 +23,7 @@ URL_REPO=https://github.com/matomo-org/matomo.git
 
 LOCAL_REPO="matomo_last_version_git"
 LOCAL_ARCH="archives"
+CACHE_DIRECTORY="/srv/matomo-cache/"
```

Comment:
Could this be moved into `/tmp/matomo-build-cache` please? `/srv` is writable only to `root`.

Reply:
This was kind of intentional. If Matt reboots his computer between two releases (which seems quite likely) the `/tmp/` will get cleared and the whole benefit of speeding up the next release is gone.